bin/setup.bat

```diff
@@ -40,8 +40,9 @@ if not "%JAVA_TOOL_OPTIONS%" == "" (
 rem JAVA_OPTS is not a built-in JVM mechanism but some people think it is so we
 rem warn them that we are not observing the value of %JAVA_OPTS%
 if not "%JAVA_OPTS%" == "" (
-  echo|set /p="warning: ignoring JAVA_OPTS=%JAVA_OPTS%; "
```

Comment:
I think the original intent behind the unusual syntax here was to preserve [not adding a newline](https://github.com/elastic/logstash/blob/master/bin/logstash.lib.sh#L80). By adding a dummy variable [as shown here](https://stackoverflow.com/questions/7105433/windows-batch-echo-without-new-line/11336754#11336754), it should be possible to keep the formatting consistent.

Reply:
yeah, the intent was to display the warning on a single line so I suggest here 
```
if not "%JAVA_OPTS%" == "" (
  echo warning: ignoring JAVA_OPTS=%JAVA_OPTS%; pass JVM parameters via LS_JAVA_OPTS
)

```